### ISSUE TITLE
Null check operator used on a null value

### ISSUE THREAD

**Issue by proficientdesigners** [sunarya-thito/shadcn_flutter#150]
Hi once again. 👋🏽

---

<img width="947" alt="image" src="https://github.com/user-attachments/assets/203174c9-3b28-4170-8b04-f2812eba151d">

---

Today after the whole migration process from `MaterialApp` theme to `ShadcnApp` while i tried to take a release of app bundle i got the above error. And it took me many hours to find from where the error occurs. Which is the usage of `TextField` widget. I even first thought there is some error on the` flutter upgrade` but it isn't.

Because i even tried the widget from fresh flutter project but the same issue happened.

Is there any quick solution for this ?

**Comment by youzhonghui** [comment]
It seems that using TextField from material or other libraries can bypass this issue.

eg.

import 'package:flutter/material.dart' as mt;

and use

mt.TextField() to replace shadcn_flutter's TextField

**Comment by proficientdesigners** [comment]
Yes, I tried that and of course it works. But there is no point of using this package if I do that, right? 🙂. Anyway, Thanks for the response. Much appreciated.

**Comment by KRTirtho** [comment]
Duplicate of #146
